File: algorithms/4.py
class Solution:
    def findMedianSortedArrays(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: float
        """
        L=nums1+nums2
        L.sort()
        
        lon=len(L)
        if lon%2 == 1:
            return L[lon//2]*1.0
        else:
            return (L[lon//2]+L[lon//2-1])/2
        
        
        

File: algorithms/4.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        
        int L[5000];
        int p=0;
        for(auto i:nums1){
            L[p]=i;
            p++;
        }
        for(auto i:nums2){
            L[p]=i;
            p++;
        }
        sort(L,L+p);
        if(p%2==1){
            return double(L[p/2]);
        }
        else{
            return (L[p/2]+L[p/2-1])/2.0;
        }
        
        
    }
};
